README.md

```diff
@@ -5,6 +5,12 @@ This action evaluate Tryber projects with [React Scripts](https://www.npmjs.com/
 
 ## Inputs
 
+- `pr_author_username`
```

Comment:
Agora que temos um input obrigatório, seria interessante atualizar em como consumir a action com esse input, nas seções [`Usage example`](https://github.com/betrybe/react-scripts-evaluator-action#usage-example) e [`How to get result output`](https://github.com/betrybe/react-scripts-evaluator-action#how-to-get-result-output), não?